user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/Interacting_with_SRM_using_code.md

```diff
@@ -10,3 +10,6 @@ Before you interact with SRM using code, you will often need to [find the object
 
 > [!TIP]
 > For more information about the types of scripts that are supported in the SRM framework, see [SRM scripting](xref:srm_scripting).
+
+> [!NOTE]
+> Since [Feature release 2.0.1](xref:SRM_2.0.1) of the SRM framework, the [SRM Dev Pack](https://www.nuget.org/packages/Skyline.DataMiner.Core.SRM) is available, which allows you to easily add the required dependencies to develop these scripts. For details on how to install the NuGet on new or existing scripts, see [Updating the SRM namespaces](xref:srm_scripting_devpack).
```

Comment:
... see [SRM scripting using the SRM Dev Pack]...

Same for the other notes (except for the one on SRM_2.0.1.md)

Reply:
Corrected!